core/views.py

```diff
@@ -766,34 +801,3 @@ def support_request(request):
                 'supportform': supportform
             }
     return HttpResponse(template.render(context, request))
-
-def signup(request):
```

Comment:
for some reason an extra signup function was in the master branch. that probably was my bad. I removed it in this branch tho